### PR TITLE
libvirt_xml.vm_xml: Added function to get cpu topology

### PR DESCRIPTION
**Pull request by sathnaga**
Added function which returns the cpu topology as dict
for example:
cputopology content in vm xml:
` <topology sockets='1' cores='4' threads='8'/>`

function returns:
{'cores': '4', 'threads': '8', 'sockets': '1'}

### CODE REVIEW

**Comment by sathnaga** [comment]
@waynesun09 @will-Do could you please merge if fine, Thanks in advance :-)

File: virttest/libvirt_xml/vm_xml.py [hunk 1]
```diff
@@ -727,6 +727,22 @@ def check_cpu_mode(mode):
             raise xcepts.LibvirtXMLError(
                 "The cpu mode '%s' is invalid!" % mode)
 
+    def get_cpu_topology(self):
+        """
+        Return cpu topology dict
+        """
+        topology = {}
+        xmltreefile = self.__dict_get__('xml')
```
Comment (waynesun09) [inline virttest/libvirt_xml/vm_xml.py]: topology is element of cpu xml, so you could get it with
vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name, virsh_instance=virsh_instance)
topology = vmxml.cpu.topology

as cpu could be without topology setting, so you need use try..except to return None if not found

**Comment by sathnaga** [comment]
@waynesun09 Thanks Updated the commit with comments

**Comment by balamuruhans** [comment]
ACK, Tested with PR https://github.com/autotest/tp-libvirt/pull/856

**Comment by will-Do** [comment]
LGTM

**Pull request merged**